user-images/ea-hub/Dockerfile

```diff
@@ -4,6 +4,10 @@ FROM earthlab/earth-analytics-python-env:ed6ee91
 # validate extension, but you can't just install one. You need to install
 # them all and then disable the ones you don't want. Hmmm, ok.
 
+# setup nbgitpuller to sync files
+RUN conda install -c conda-forge nbgitpuller
```

Comment:
install from conda-forge and enable. NOTE: i haven't looked to see if functionality has changed!!